### DISCUSSION TITLE
No DROPLET snapshot found with name....

### DISCUSSION THREAD

**Discussion by oguruma1218** [digitalocean/supabase-on-do#38] [Q&A]
When running terraform plan, I get the error: `Error: No DROPLET snapshot found with name with data.digitalocean_droplet_snapshot.supabase,  on droplet.tf line 1, in data "digitalocean_droplet_snapshot" "supabase": 1: data "digitalocean_droplet_snapshot" "supabase"`

I've never used terraform before, so I'm sure I'm doing something wrong. 

What exactly does the first line "data.digitalocean_droplet_snapshot" "supabase" represent?

**Comment by timothymamo** [comment]
When you run the `terraform plan/apply` command terraform looks for the snapshot created with packer in the [previous step](https://docs.digitalocean.com/developer-center/hosting-supabase-on-digitalocean/#build-image-using-packer). You should also be able to see your snapshot within the DigitalOcean cloud UI by clicking on the `Images` menu in the left navigation bar.

If terraform cannot find this image it will show you the error you are getting. Did you run the packer command successfully? Can you see a snapshot within the UI? It will have the following format `supabase-YYYYMMDDHHMMSS` with the date format of when you created the snapshot.

**Comment by oguruma1218** [comment]
Does it have to be a Snapshot for a droplet that actually exists? Or could I create the droplet, make the snapshot, then delete that droplet (and still run the terraform based on it)?

**Comment by timothymamo** [comment]
When you run the packer command a droplet is created which is what the snapshot is created from. The droplet is subsequently deleted when once the snapshot is created.

Did you run the packer commands as explained in the blog post and the README file?
